test/NuGet.Clients.Tests/NuGet.CommandLine.Test/CommandLineParserTests.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Xunit;
+using NuGet.Commands;
+using System;
+using Moq;
+using System.Reflection;
+
+namespace NuGet.CommandLine.Test
+{
+    public class CommandLineParserTests
+    {
+        private Mock<CommandManager> _mockCommandManager = new Mock<CommandManager>();
```

Comment:
When testing, mock the dependency by interface type, not concrete class:

```suggestion
        private Mock<ICommandManager> _mockCommandManager = new Mock<ICommandManager>();
```